frontend/src/constants/app.ts

```diff
@@ -13,3 +13,6 @@ export const SIGNOZ_UPGRADE_PLAN_URL =
 	'https://upgrade.signoz.io/upgrade-from-app';
 
 export const DASHBOARD_TIME_IN_DURATION = 'refreshInterval';
+
+export const QUERY_BUILDER_URL =
```

Comment:
Hi @palashgdev, we moved the URL to constants but are not sure if this is the right file to move to

Reply:
@gitstart just move under Explorer Card constant not over global one